pkgs/by-name/pi/pid-fan-controller/package.nix

```
@@ -0,0 +1,26 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  lib,
+}:
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage {
```

**non-blocking suggestion:** Replace this with a `let...in` statement and `inherit` as needed. This is in accordance with best practices from the [Nix documentation](https://nix.dev/guides/best-practices.html) and #315337.

That said, since it's such a simple package, feel free to keep `rec`.